src/components/AddFundsInterstitial.js

```diff
@@ -219,7 +220,7 @@ const AddFundsInterstitial = ({ network }) => {
   }, [navigate, isDamaged]);
 
   return (
-    <Container>
+    <Container isSmallScreen={isSmallPhone || isTinyPhone}>
```

Comment:
```suggestion
    <Container isSmallScreen={isSmallPhone}>
```

It's not possible that the phone is tiny, but not small